modin/tests/pandas/test_io.py

```diff
@@ -3461,3 +3461,20 @@ def test_from_dask():
 
     result_df = from_dask(dask_df)
     df_equals(result_df, modin_df)
+
+
+@pytest.mark.skipif(
+    condition=Engine.get() not in ("Ray", "Dask", "Unidist"),
```

Comment:
Would it be more correct to limit it not by engines, but by storage format: `pandas`?

Reply:
PandasOnPython wouldn't work. Let's leave as is.

Reply:
> PandasOnPython wouldn't work.

As far as I can see, there are no restrictions on its operation. We just need to add essentially the same code as for the other engines.